README.md

```diff
@@ -1,210 +1,305 @@
 --------------------------------------------------------------------------------
 
-Adjusting to compact keyboard layouts isn’t easy, but Arsenik is designed for
-a step-by-step approach.
+Choose which Arsenik features to use from the following options:
+
+
+### 1. Angle mod
+
+On an ISO keyboard, it permuts the extra down left key to ease the angle on your
+left wrist when typing.
 
-### 1. Supercharge Your Thumbs
+![Angle mod](./img/angle_mod.svg)
+
+
+### 2. Supercharge your thumbs with mod-taps
+
+#### First: layer-taps
 
 If you’re new to mod-taps, we suggest to start by adding the “layer-tap” option
 where only the thumbs are affected:
 
 - the left thumb key remains a <kbd>Cmd</kbd> or <kbd>Alt</kbd> key when held,
 but emits a <kbd>Backspace</kbd> when tapped;
-- the right thumb key brings the <kbd>Symbols</kbd> layer when held (similar to
-an <kbd>AltGr</kbd> key), and emits <kbd>Return</kbd> when tapped;
-- the spacebar brings the <kbd>Navigation</kbd> layer when held.
+- the right thumb key brings the <kbd>Symbols</kbd> layer when held (in blue)
+— where all programming symbols are arranged for comfort and efficiency — and
+emits <kbd>Return</kbd> when tapped;
+- the spacebar brings the <kbd>Navigation</kbd> layer when held (in orange).
 
-![alt, navigation and sym layers under the thumbs](img/base_easy.svg)
+![alt, navigation and sym layers under the thumbs](./img/layer_taps.svg)
 
 Having <kbd>Backspace</kbd> and <kbd>Enter</kbd> under the thumbs is enough to
-reduce the pinky fatigue very significantly. And using the <kbd>Symbol</kbd>
-and <kbd>Navigation</kbd> layer further reduces hand and finger movements.
+reduce the pinky fatigue very significantly. And using the <kbd>Symbols</kbd>
+and <kbd>Navigation</kbd> layers further reduces hand and finger movements.
+
 
-### 2. Enable the Home-Row Mods
+#### Next level: enable the Home Row Mods
 
-When you are familiar with mod-taps, it’s time to enable them on the homerow
-with the “hrm” variants:
+When you are familiar with mod-taps, it’s time to enable them on the home row
+with the “HRM” variants:
 
 - <kbd>FDS</kbd> and <kbd>JKL</kbd> become <kbd>Ctrl</kbd>, <kbd>Alt</kbd>,
 <kbd>Super</kbd> when held long enough;
 - the left thumb key can now emit a <kbd>Shift</kbd> rather than <kbd>Alt</kbd>
 when held.
 
-![homerow mods on SDF keys](img/base_hrm.svg)
+![home row mods on SDF keys](./img/hrm.svg)
+
+This is a very basic variant of the [Miryoku] principle: one layer on each
+thumb key, and symmetrical modifiers on the home row.
+
+
+### 3. Symbols layer
 
-This is a very basic variant of the [Miryoku][1] principle: one layer on each
-thumb key, and symmetrical modifiers on the homerow.
+For the <kbd>Symbols</kbd> layer you can keep <kbd>AltGr</kbd> as-is. It is
+useful for keyboard layouts that rely heavily on the <kbd>AltGr</kbd> key.
 
-### 3. Spice It Up
+But the real fun (especially for programmers) happens when we enable the “One 
+Dead Key” (= 1DK) programmation layer!
```

Comment:
I’m not sure “1DK” is relevant in the context of the Symbols layer. (And yes I’m aware this is the org name I’ve chosen…)

We usually refer to this layer as “Lafayette” but I guess “Arsenik” would be okay too.
@Nuclear-Squid wdyt?

Reply:
1DK layer is really misleading for layouts in the Lafayette Family, on top of it not being relevant in this context so I agree on *not* using this.

As for Arsenik / Selenium / Lafayette, I’ll be a pedant and say that Lafayette makes more sense : This “prog layer” only serves to emulate what we designed for the Lafayette layouts. It’s not an Arsenik / Selenium innovation, it’s a Lafayette innovation (just like how Selenium has a “vim nav” layer : we didn’t invent arrows on `hjkl`, but we are emulating it because it’s really cool).

Yes, the “1DFH philosophy” is shared with Arsenik, but so is a *lot* of design decisions with the Lafayette layouts, and this prog layer is *exactly the same* across all of them. If we had a layer to emulate Colemak on Qwerty (for instance), we would have called it a “Colemak layer” without thinking, and I feel “Lafayette” makes sense for the same reason.